unittest/main.cpp

```diff
@@ -30,7 +29,6 @@ unsigned suite_hashutil(unsigned);
 unsigned suite_legacy_util(unsigned);
 
 const suite_fn k_legacy_suites[] = {
-  &suite_args,
```

Comment:
I don't want to remove the legacy args suite as long as the legacy args API is still in use. (And when we do we should remove `unittest/test_legacy_args.cpp` as well.)

What build errors did you get with Clang 10? I can build with Clang 10 without problems.

Reply:
Hups. I missed adding the new file test_legacy_args.cpp